moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -100,8 +100,10 @@ bool ButterworthFilterPlugin::initialize(rclcpp::Node::SharedPtr node, moveit::c
 {
   node_ = node;
   num_joints_ = num_joints;
-  auto param_listener = std::make_unique<online_signal_smoothing::ParamListener>(node_);
-  auto filter_coeff = param_listener->get_params().butterworth_filter_coeff;
+  {
+    online_signal_smoothing::ParamListener param_listener(node_);
+    filter_coeff = param_listener.get_params().butterworth_filter_coeff;
```

Comment:
You need to initialize this param outside the scope, otherwise it is lost at the end of the scope and currently this variable is not initialized at all

Reply:
duh. Done.